### DISCUSSION TITLE
Using data dir based on freedesktop specs

### DISCUSSION THREAD

**Discussion by andy5995** [KaruroChori/vs-fltk#62] [General]
Instead of using `~/.vs-fltk`, it might be worth considering checking for $XDG_DATA_HOME first, and if that doesn't exist, use `~/.local/share/vs-fltk` 

[XDG Base Directory Specification](https://specifications.freedesktop.org/basedir-spec/latest/)

**Comment by KaruroChori** [comment]
I am not too convinced about this one.  
There are many applications following the freedesktop specs... and many more which do not.

For example, on the laptop I am writing from, `bun`, `cargo`, `dillo`, `mozilla`, `npm`, `platformio`, `renpy`, `wine`, `thunderbird`, `vscode-oss`, they are all located in home within their folder. They don't split their files between `.config` and `.local/share`, they keep all user related data in one place.

Only `flatpak`, `gambas3`, `godot`, `nano` & `Mousepad` follow this convention. 

So, even if it is a standardization effort, its adoption is somewhat limited from what I can tell and it would require me to split user's file in multiple places.
